android/app/src/main/java/org/fossasia/openevent/fragments/SpeakersListFragment.java

```diff
@@ -118,6 +120,7 @@ public void onRefresh() {
                     public void accept(@NonNull List<Speaker> speakers) throws Exception {
                         mSpeakers.clear();
                         mSpeakers.addAll(speakers);
+                        array = DbSingleton.getInstance().getCount(mSpeakers).get(0);
```

Comment:
Just count them here instead of going to the database again

Here : 20 ms
Database : 750 + 20 ms

(Figuratively)